Cart-management/cart-management.css

```diff
@@ -0,0 +1,264 @@
+@import url(/Utility/utility.css);
+
+.icon {
+    color: var(--color-light-gray);
+    font-size: 2rem;
+}
+
+.inner-icon {
+    color: var(--color-light-gray);
+    font-size: 1.3rem;
+    padding: 1rem;
+}
+
+.input-box button {
+    padding: 1.2rem;
+    outline: none;
+}
+
+button {
+    border-radius: 0;
+    box-shadow: var(--shadow-mild);
+}
+
+.button-icons {
+    font-size: 1rem;
+}
+
+nav .actions {
+    gap: 2rem;
+    padding: 0 2%;
+}
+
+nav .input-box {
+    min-width:30%;
+}
+
+nav .input-box input {
+    padding: 1rem 0;
+}
+
+nav h3 {
+    padding: 2%;
+}
+
+.content li,
+.actions {
+    margin: 1rem 0;
+}
+
+.input-box input {
+    padding: 12px;
+}
+
+.input-box {
+    padding: 1rem 0;
+}
+
+.input-icon-container {
+    flex-wrap: nowrap;
+    border-radius: 0;
+}
+
+nav input::placeholder {
+    font-size: 1.4rem;
+}
+
+.card {
+    box-shadow: var(--shadow-mild);
+    margin:1rem;
+}
+
+.main-wrapper h6{
+    padding:2rem
+}
+
+.content {
+    font-size:13px;
+}
+
+.content-wrapper {
+    gap: 10%;
+}
+
+.main-wrapper {
+    width: 90%;
+    margin: 0 auto;
+}
+
+.content h3, .content h4 {
+    font-size: 170%;
+}
+
+.card .image-container {
+    width: 40%;
+}
+
+.image-container {
+    width: fit-content;
+}
+
+.star-img {
+    color: orange;
+}
+
+.quantity-counter {
+    gap: 1rem;
+    padding: 8px;
+}
+
+.quantity-btn {
+    padding: 8px;
+    font-size: 1.1rem;
+    box-shadow: none;
+    border-radius: var(--button-border);
+}
+
+.quantity-btn:hover {
+    box-shadow: none;
+} 
+
+.stock-quantity p {
+    font-size: 1.2rem;
+}
+
+.checkout-card .card{
+    padding: 3rem 1rem 1rem 1rem;
+}
+
+.particulars {
+    border-bottom: 1px solid;
+    padding:1rem 0;
+}
+
+.checkout-card p {
+    font-size:1.1rem;
+}
+
+.bill-total p{
+    padding-top: 1rem ;
+    font-size:1.3rem;
+}
+
+.checkout-card .primary-button {
+    font-size: 1.2rem;
+}
+
+.checkout-card .outline-secondary {
+    border: none;
+    font-size: 1.1rem;
+}
+
+.payment-partners svg {
+    width: 3rem
+}
+
+.terms-and-policy p{
+    padding: 1rem 0;
+    font-size: 1rem;
+}
+
+.terms-and-policy span {
+    border-bottom: 1px solid;
+}
+
+footer {
+    background-color: var(--color-secondary);
+    color: var(--color-white);
+    padding: 2rem;
+    flex-wrap: wrap;
+}
+
+footer h6 {
+    padding: 1rem 0;
+}
+
+li {
+    list-style: none;
+}
```

Comment:
Can store this piece of code in utilities.css if it is used frequently

Reply:
yes, I'll do that